content/team.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Team
+layout: page
+---
+
+# PyCon India 2021 Team
+
+The conference wouldn't have concluded on a high note much less even started planning without the constant efforts of countless volunteers which were part of this year's workgroups. The following is a list of individuals that came together to work for 10+ months in order to make the online conference this year a great experience for everyone attending.
+
+![A world map with names of pycon india volunteers](https://in.pycon.org/2021/images/team-world-map.png) 
+
+- Abhishek Mishra
+- Abhishek Upperwal
+- Akshay Gaikwad
+- Anirudh Menon
+- Aravind Sekar
+- Arun
+- Bharat
+- Bhavin Gandhi
+- Chandan Kumar
+- Darshna Das
+- Divya Goswami
+- Duranjai Malik
+- Farhaan Bukhsh
+- Gajendra Deshpande
+- Garvit Kulshreshtha
+- Girish Joshi
+- Hari priya
+- Harsh Mittal
+- Jatin Goel
+- Kalyan Prasad
+- Kumar Anirudha
+- Laisha Wadhwa
+- Manaswini Das
+- Megha
+- Minal Patil
+- Nabarun Pal
+- Nikhil Maan
+- Pradhvan Bisht
+- Praveen Patil
+- Priyanshu Kumar
+- Rahul B
+- Rahul Kumaresan
+- Rayan Das
+- Ritesh Agrawal
+- Rohan Samanta
+- Sanchit Balchandani
+- Sandeep Choudhary
+- Saptak S
```

Comment:
Also Saptak Sengupta if we're using full name of everyone.

Reply:
Saptak has asked to keep it Saptak S